src/ukify/ukify.py

```diff
@@ -1114,27 +1150,57 @@ def config_example():
             yield f'{key} = {value}'
 
 
-def create_parser():
+def add_items_to_parser(conf_items, p):
+    for item in conf_items:
+        item.add_to(p)
+
+    # Suppress printing of usage synopsis on errors
+    p.error = lambda message: p.exit(2, f'{p.prog}: error: {message}\n')
+
+
+def create_main_parser():
     p = argparse.ArgumentParser(
-        description='Build and sign Unified Kernel Images',
+        description='Create and display information of Unified Kernel Images',
         allow_abbrev=False,
-        usage='''\
-ukify [options…] [LINUX INITRD…]
-''',
-        epilog='\n  '.join(('config file:', *config_example())),
+        usage = """\nukify [options…] [actions]""",
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
 
-    for item in CONFIG_ITEMS:
-        item.add_to(p)
+    add_items_to_parser(CONFIG_ITEMS_COMMON, p)
 
-    # Suppress printing of usage synopsis on errors
-    p.error = lambda message: p.exit(2, f'{p.prog}: error: {message}\n')
+    return p
```

Comment:
so not sure i follow here, but this is a compat break? i.e. we released ukify with systemd 253 already, and command lines lacking "create" from back then will now fail once this is merged, no? that's not ok.

Reply:
didn't we say it was experimental?

Reply:
we did not. and even if we had done so, not an excuse to go overboard with changes.

Reply:
so I tried to do what you suggested, basically default to 'create' (or 'make', same for me) when no verb is given. Turns out this is a nightmare to do with python argparse, because it will get very confused when a parameter without `--` is given. For example, `ukify --output` will nicely default to `ukify make --output`, but `ukify vmlinux --output` will crash because `vmlinux` will be compared to `[make, display]` subparsers and won't be recognized

Reply:
Alternatively we can do `--linux=vmlinux` but that is also a compatibility breakage.

Reply:
third alternative is to parse the stuff ourselves first and then give it to argparse. But it won't be pretty

Reply:
Maybe a bit hacky, but a solution would probably be
```python
    if sys.argv[1] not in ("create", "display"):
        sys.argv.insert(1, "create")
```
in `create_parser` right before returning the created parser. It's a bit ugly because now the function has a side effect and mutates argv, but it does what is asked.

Reply:
slightly more complex to make tests happy, but yeah that's it:
``` python
    if args is None:
        args = sys.argv[1:]
    if args[0] not in ACCEPTED_COMMANDS:
        args.insert(0, MAKE_COMMAND)
  ```
  I'll push this fix, because I don't see any other way to do this. Hopefully nobody will complain :crossed_fingers: 
  Honestly using verbs makes `ukify` much more organized, especially when adding the `display` function.